cli/command/service/logs.go

```diff
@@ -92,7 +92,7 @@ func runLogs(dockerCli *command.DockerCli, opts *logsOptions) error {
 		if !client.IsErrServiceNotFound(err) {
 			return err
 		}
-		task, _, err := cli.TaskInspectWithRaw(ctx, opts.target)
```

Comment:
ya, I had the same thought, but didn't want to do it here. I know I reviewed that PR, and I think I noticed it at the time, but convinced myself that the error wouldn't be an issue.

How about I open an issue for it to track it, and we merge this?